Team417_PowerPlay/src/main/java/org/firstinspires/ftc/team417_PowerPlay/drive/SampleMecanumDrive.java

```diff
@@ -58,8 +58,7 @@ public class SampleMecanumDrive extends MecanumDrive {
 
     public static PIDCoefficients TRANSLATIONAL_PID = new PIDCoefficients(0.7, 0, 0);
     public static PIDCoefficients HEADING_PID = new PIDCoefficients(2, 0, 0);
-//😃 -> shift + insert woahh
-    public static double LATERAL_MULTIPLIER = 1;
+    public static double LATERAL_MULTIPLIER = 60.0 / 46.319;
```

Comment:
Where are these magic numbers coming from?

Reply:
It came from the energy deficiency that we get while strafing